project-base/storefront/components/Blocks/Skeleton/SkeletonPageBlogCategory.tsx

```diff
@@ -1,25 +1,27 @@
 import { SkeletonModuleArticleBlog } from './SkeletonModuleArticleBlog';
 import { SkeletonModuleBreadcrumbs } from './SkeletonModuleBreadcrumbs';
 import { Webline } from 'components/Layout/Webline/Webline';
-import { DEFAULT_PAGE_SIZE } from 'config/constants';
+import { DEFAULT_BLOG_PAGE_SIZE } from 'config/constants';
 import Skeleton from 'react-loading-skeleton';
 import { createEmptyArray } from 'utils/arrays/createEmptyArray';
 
 export const SkeletonPageBlogCategory: FC = () => (
     <Webline>
         <SkeletonModuleBreadcrumbs count={2} />
         <div className="scroll-mt-5">
-            <div className="mb-16 flex flex-col vl:flex-row">
-                <div className="order-2 mb-16 flex w-full flex-col vl:order-1 vl:flex-1">
-                    <Skeleton className="mb-3 h-8 w-3/5" />
-                    <div className="mb-16 flex flex-col gap-8">
-                        {createEmptyArray(DEFAULT_PAGE_SIZE).map((_, index) => (
+            <div className="mb-6 md:mb-10">
+                <Skeleton className="h-[196px] w-full rounded-xl" />
+            </div>
+            <div className="mb-[60px] flex flex-col gap-3 md:gap-10 vl:flex-row xl:gap-[116px]">
```

Comment:
just double check if `px` is necessary, otherwise is ok

Reply:
fixed